src/routes/klageskjema/begrunnelse/begrunnelse.tsx

```diff
@@ -54,6 +54,7 @@ const Begrunnelse = ({ klage }: Props) => {
     const [attachmentsLoading, setAttachmentsLoading] = useState<boolean>(false);
     const [attachmentError, setAttachmentError] = useState<string | null>(null);
     const [submitted, setSubmitted] = useState<boolean>(false);
+    const [autosaveStatus, setAutosaveStatus] = useState<number>();
```

Comment:
Bruk `AutosaveStatus` som type i stedet for `number`, da får vi bedre hjelp av TS :)